kolibri/plugins/management/assets/src/vue/user-page/user-create-modal.vue

```diff
@@ -1,46 +1,49 @@
 <template>
 
   <div class="user-creation-modal">
-    <modal @open="clear" v-ref:modal btntext="Add New">
-
-      <h1 slot="header" class="header">Add New Account</h1>
+    <modal @open="clear" v-ref:modal title="Add New Account" btntext="Add New">
 
       <div @keyup.enter="createNewUser" slot="body">
 
-        <div class="user-field">
-          <label for="name">Name</label>
-          <input @focus="clearErrorMessage" type="text" class="add-form" id="name" autocomplete="name"  autofocus="true" required v-model="full_name">
-        </div>
-
-        <div class="user-field">
-          <label for="username">Username</label>
-          <input @focus="clearErrorMessage" type="text" class="add-form" autocomplete="username" id="username" required v-model="username">
-        </div>
-
-        <div class="user-field">
-          <label for="password">Password</label>
-          <input @focus="clearErrorMessage" type="password" class="add-form" id="password" required v-model="password">
-        </div>
-
-        <div class="user-field">
-          <label for="confirm-password">Confirm Password</label>
-          <input @focus="clearErrorMessage" type="password" class="add-form" id="confirm-password" required v-model="passwordConfirm">
-        </div>
-
-        <div class="user-field">
-          <label for="user-role"><span class="visuallyhidden">User Role</span></label>
-          <select @focus="clearErrorMessage" v-model="role" id="user-role">
-          <option value="learner" selected> Learner </option>
-          <option value="admin"> Admin </option>
-          </select>
-        </div>
-
+        <!-- Fields for the user to fill out -->
+        <section class="user-fields">
```

Comment:
Eventually, we could probably do with a general purpose 'form' component, but this will suffice for now.